README.md

```diff
@@ -1 +1,106 @@
-iRODS HTTP API: https://github.com/irods/irods_client_http_api
+# iRODS Java HTTP API Wrapper
+
+Java client library designed and developed to wrap the new iRODS HTTP API. It provides java classes to cleanly interact 
+with all the functionality of the HTTP API and comes with tests and packaging.
+
+## How to Use?
+### Client
+Create a new `IrodsHttpClient` instance and pass in the base URL. 
+```java
+IrodsHttpClient client = new IrodsHttpClient("http://localhost:<port>/irods-http-api/<version>");
+```
+
+`<version>` must match the format, X.Y.Z, where X is the major version number, Y is the minor version number, and Z is the patch version number.
+
+You can only access the information endpoint without authenticating. All other operations require a token.
+
```

Comment:
This can be removed because it's demonstrated in the usage example.

Reply:
It's been removed.